.gitignore

```diff
@@ -1,3 +1,7 @@
+.build/
+.vscode/
```

Comment:
Visual Studio Code?

Reply:
Ah, I forgot. It just because I use VSCode. I am sorry.